There is CSS linter issue in CI/CD script. Please, fix the code styling.

```

Run npm run stylelint

> green-city-client@0.0.0 stylelint
> npx stylelint "src/**/*.scss"


src/app/main/component/eco-news/components/create-edit-news/create-edit-news.component.scss
  90:1  ✖  Unexpected duplicate selector ".title-description", first used at line 85  no-duplicate-selectors

✖ 1 problem (1 error, 0 warnings)

Error: Process completed with exit code 2.
```

--- a/src/app/main/component/eco-news/components/create-edit-news/create-edit-news.component.html
+++ b/src/app/main/component/eco-news/components/create-edit-news/create-edit-news.component.html
@@ -3,7 +3,9 @@
   <div class="box" *ngIf="!isPosting">
     <div class="title">
       <h2 class="title-header">{{ attributes.title | translate }}</h2>
-      <p class="title-description">{{ 'create-news.form-description' | translate }}</p>
+      <div class="title-description-wrapper">
+        <p class="title-description">{{ 'create-news.form-description' | translate }}</p>
+      </div>
     </div>
     <div class="form-container" *ngIf="form">
       <form [formGroup]="form" (ngSubmit)="onSubmit()" enctype="multipart/form-data">

--- a/src/app/main/component/eco-news/components/create-edit-news/create-edit-news.component.scss
+++ b/src/app/main/component/eco-news/components/create-edit-news/create-edit-news.component.scss
@@ -87,12 +87,6 @@ form label {
   }
 }
 
-.title-description {
-  max-width: 58%;
-  word-break: break-word;
-  white-space: normal;
-}
-
 .wrapper {
   display: grid;
   grid-template: auto auto auto / 652px auto;
@@ -198,6 +192,20 @@ textarea.ng-invalid.ng-dirty + p {
   border: 1px solid red;
 }
 
+.title-description {
+  max-width: 58%;
+  word-break: break-word;
+  white-space: normal;
+
+  @media screen and (max-width: 1023px) {
+    max-width: 75%;
+  }
+
+  @media screen and (max-width: 767px) {
+    max-width: 100%;
+  }
+}
+
 @media screen and (max-width: 1439px) {
   .box {
     width: 960px;
@@ -223,11 +231,6 @@ textarea.ng-invalid.ng-dirty + p {
       'image'
       'source';
   }
-
-  .title-description {
-    max-width: 75%;
-    white-space: normal;
-  }
 }
 
 @media screen and (max-width: 767px) {
@@ -240,11 +243,6 @@ textarea.ng-invalid.ng-dirty + p {
       font-size: 24px;
       line-height: 32px;
     }
-  }
-
-  .title-description {
-    max-width: 100%;
-    white-space: normal;
   }
 }
 

--- a/src/app/main/component/eco-news/components/create-edit-news/create-edit-news.component.scss
+++ b/src/app/main/component/eco-news/components/create-edit-news/create-edit-news.component.scss
@@ -90,6 +90,7 @@ form label {
 .title-description {
   max-width: 58%;
   word-break: break-word;
+  white-space: normal;
 }
 
 .wrapper {
@@ -221,6 +222,11 @@ textarea.ng-invalid.ng-dirty + p {
       'tags'
       'image'
       'source';
+  }
+
+  .title-description {
+    max-width: 75%;
+    white-space: normal;
   }
 }
 

--- a/src/app/main/component/eco-news/components/create-edit-news/create-edit-news.component.scss
+++ b/src/app/main/component/eco-news/components/create-edit-news/create-edit-news.component.scss
@@ -192,7 +192,7 @@ textarea.ng-invalid.ng-dirty + p {
   border: 1px solid red;
 }
 
-.title-description {
+.title-description-wrapper .title-description {
   max-width: 58%;
   word-break: break-word;
   white-space: normal;

